modules/nw-secondary-ext-gw-about.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-external-gateway.adoc
+
+:_content-type: PROCEDURE
+[id="nw-secondary-ext-gw-about_{context}"]
+= How {product-title} determines the external gateway IP address
+
+You configure a secondary external gateway with the `AdminPolicyBasedExternalRoute` custom resource from the the `k8s.ovn.org` API group. The custom resource (CR) supports static and dynamic approaches to specifying an external gateway's IP address. Each namespace that a `AdminPolicyBasedExternalRoute` CR targets cannot be selected by any other `AdminPolicyBasedExternalRoute` CR. A namespace cannot have concurrent secondary external gateways.
+
```

Comment:
Maybe it's just my rusty English or confusion on my part, if we deploy a static route with dual gateways as described later in document under the static route config section/example: 
    static:
    - ip: "172.18.0.8" 
    - ip: "172.18.0.9"

It sort of contradicts the last sentence "A namespace cannot have concurrent secondary external gateways."  As the targeted namespace of our example would indeed have two concurrent secondary external gateways 172.18.0.8 and 172.18.0.9.  If only I find this a bit confusing feel free to ignore.